views/components/Sidebar.jsx

```diff
@@ -29,36 +30,14 @@ const Sidebar = ({
       alert(error.message)
     }
   }
-
   return (
     <div className={classes.sideBar}>
-      <Hidden mdUp>
-        <Drawer
-          variant="temporary"
-          anchor={theme.direction === 'rtl' ? 'right' : 'left'}
-          open={mobileOpen}
-          onClose={handleDrawerToggle}
-          classes={{
-            paper: classes.drawerPaper,
-          }}
-          ModalProps={{
-            keepMounted: true, // Better open performance on mobile.
-          }}
-        >
-          <DrawerComponent
-            classes={classes}
-            onLogout={handleLogout}
-            totalStudies={totalStudies}
-            totalSubjects={totalSubjects}
-            totalDays={totalDays}
-            user={user}
-          />
-        </Drawer>
-      </Hidden>
-      <Hidden smDown implementation="css">
+      <Hidden>
```

Comment:
if this is hidden do we need it?

Reply:
Good find. We don't.
[3589154](https://github.com/AMP-SCZ/dpdash/pull/352/commits/3589154956739d104ec9936752a25f11f86f0750)